src/mapGenerator/Map.cpp

```diff
@@ -23,18 +23,12 @@ Map::Map() : width(0), height(0)
     
 }
 
-Map::Map(unsigned int width,
-         unsigned int height,
+Map::Map(unsigned width,
+         unsigned height,
          const std::string& name,
-         const std::string& author) : width(width), height(height), name(name), author(author), positions(MAX_PLAYERS)
+         const std::string& author) : width(width), height(height), name(name), author(author), positions(MAX_PLAYERS, Point<uint16_t>::Invalid())
 {
-    const unsigned int size = (unsigned int)width * height;
-
-    for (unsigned int i = 0; i < positions.size(); i++)
-    {
-        positions[i].x = 0xFF;
-        positions[i].y = 0xFF;
-    }
+    const unsigned size = (unsigned)width * height;
```

Comment:
Note: Unnecessary cast. And try to use C++ casts ( `static_cast` and family) Can be done in a follow up though